standard-format.json

```diff
@@ -14,6 +14,11 @@
       "en-US": "Spawn a Botchling or a Lubberkin.",
       "etc.": "etc.",
     },
+    "infoRaw": {
+      "de-DE": "<keyword=spawn>Generiere</keyword> einen Fehlgeborenen oder einen T\u00f6lpelbold.",
```

Comment:
Just for completeness, in the raw files, the `"` is not present:

`"tooltips";"tooltip_0030_description";"<b><keyword=deploy>Deploy:</keyword></b> <keyword=spawn>Spawn</keyword> a Unicorn or a Chironex."`